docs/source/onnxruntime/usage_guides/gpu.mdx

```diff
@@ -314,25 +316,20 @@ For example, for text generation, the engine can be built with:
 >>> provider_options = {
 ...     "trt_engine_cache_enable": True,
 ...     "trt_engine_cache_path": "tmp/trt_cache_gpt2_example"
+...     "trt_profile_min_shapes": "input_ids:1x1,attention_mask:1x1,position_ids:1x1",
+...     "trt_profile_opt_shapes": "input_ids:1x1,attention_mask:1x1,position_ids:1x1",
+...     "trt_profile_max_shapes": "input_ids:1x64,attention_mask:1x64,position_ids:1x64",
 ... }
 
 >>> ort_model = ORTModelForCausalLM.from_pretrained(
-...     "optimum/gpt2",
+...     "gpt2",
+...     export=True,
 ...     use_cache=False,
+...     use_merged=False,
```

Comment:
I am curious about these args. Is there an issue about `optimum/gpt2` (guessing no `position_ids` input?)? Why `use_merged=False`?

Reply:
okay it seems that `optimum/gpt2` works as well, I reverted to the original